_harp/services/ratecard.md

```diff
@@ -0,0 +1,16 @@
+# Rate card
+
+|    |    |
+|----|---:|
+|Graduate developer team|£500/week|
+|Graduate developer - follow-up work|£80/day|
+|Apprentice developer|£100/day|
```

Comment:
What happened to people setting their own rates? Surely this should _at most_ be an _indicative_ rate card with prices _around_ this mark until fixed prices are agreed with the coop?